pyproject.toml

```diff
@@ -181,9 +172,6 @@ types-setuptools = "^65.1.0"
 memory-profiler = "^0.60.0"
 psutil = "^5.8.0"
 mypy-extensions = "^0.4.3"
-# newer versions of sanic testing are not compatible 
-# with our version of sanic. sanic-testing 22.9.0 introduced
-# changes that don't work for our sanic version
```

Comment:
Why was this comment removed?

Reply:
@ancalita I've added it back i think comments were cleaned up with the new changes made after m1 support.